src/components/AnonymousChat/ChatDetailHeader.tsx

```diff
@@ -47,7 +47,7 @@ const styles = StyleSheet.create({
   },
   text: {
     fontFamily: fonts.inter[600],
-    color: COLORS.white,
+    color: COLORS.white2,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text color has been updated from `COLORS.white` to `COLORS.white2`. Make sure this change doesn't affect readability against its background.
```diff
-    color: COLORS.white,
+    color: COLORS.white2,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->